### PR TITLE
Added test for deleted links

### PR DESCRIPTION
**Pull request by Alopalao**
Related PR [#179](https://github.com/kytos-ng/topology/pull/179)

### Summary

Added test for API endpoint that deletes link.

### Result
```
+ python3 -m pytest tests/ -k test_130_delete_link --reruns 0 -r fEr
============================= test session starts ==============================
platform linux -- Python 3.9.2, pytest-7.2.0, pluggy-1.3.0
rootdir: /tests
plugins: rerunfailures-10.2, timeout-2.1.0, anyio-3.6.2
collected 255 items / 254 deselected / 1 selected

tests/test_e2e_05_topology.py .                                          [100%]
```

### CODE REVIEW

File: tests/test_e2e_05_topology.py [hunk 1]
```diff
@@ -635,6 +635,65 @@ def test_120_removing_link_metadata_persistent(self):
...
+            assert response.status_code == 200, response.text
+
+        self.restart()
+
+        # Get the link_id
+        api_url = KYTOS_API + '/topology/v3/links'
+        response = requests.get(api_url)
+        assert response.status_code == 200
+        data = response.json()
+        print("DATA -> ", data)
```
Comment (viniarck) [inline tests/test_e2e_05_topology.py:662]: Looks like this `print` was just temporary if you could remove it.

**Review by viniarck — COMMENTED** [review]
LGTM. Did you also run this test locally? I'm just awaiting for this confirmation to leave it pre-approved.

File: tests/test_e2e_05_topology.py (hunk 1 above)
Comment (Alopalao) [reply tests/test_e2e_05_topology.py]: Deleted

**Review by viniarck — APPROVED** [review]
LGTM. Leaving it pre-approved. Once https://github.com/kytos-ng/topology/pull/179 is merged, let's merged this one here too.

**Pull request merged**